### PR TITLE
cleanup unused code

### PR DESCRIPTION
**Pull request by rymurr**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie-demos/162)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,43 @@
...
+* [Spark and Delta](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-delta-demo-nba.ipynb)
+* [Flink and Iceberg](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-flink-demo-nba.ipynb)
 
-* Nessie-Iceberg-NBA Jupyter Notebook
-    * [Google Colaboratory URL](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
-    * [Binder (mybinder.org)](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb)
-* Nessie-Deltalake-NBA Jupyter Notebook
-  * [Google Colaboratory URL](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-delta-demo-nba.ipynb)
-  * [Binder (mybinder.org)](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-delta-demo-nba.ipynb)
+The are automatically rebuilt every time we push to master. They are unit tested using `testbook` library to ensure we get
```
Comment (omarsmak) [inline README.md:9]: I guess the branch here is `main` not `master` ?

File: README.md [hunk 2]
```diff
@@ -1,76 +1,43 @@
...
-Users basically just provide a URL to a `.ipynb` file, which is then loaded. The runtime
-environment is a container/image that is pre-defined by Google. Changing the base operating
-system and such is not possible. Bumping the pre-installed Python dependencies is not feasible.
+Only has to be updated in `binder/requirements.txt`. Currently Iceberg supports 3.0.x and 3.1.x while delta late supports
+3.1.x only.
 
-### Binder
+### Flink
 
-[Binder](https://mybinder.org) is a more customizable platform for Jupyter notebooks and
```
Comment (omarsmak) [inline README.md:60]: Do you think if it makes sense to keep the brief description of Binder for anyone who is not familiar with it (including myself) :D ?

Comment (rymurr) [reply README.md:60]: done

File: README.md (hunk 1 above)
Comment (rymurr) [reply README.md]: done

**Review by snazy — APPROVED** [review]

**Review by omarsmak — APPROVED** [review]

**Pull request merged**